src/test/java/Homework17.java

```diff
@@ -22,14 +23,21 @@ public void addSongToPlaylist(){
         clickOnSuperPlaylist();
 
         //Verify the song
-        String songFromSuperPlaylist = getSongTitleFromSuperPlaylist();
-        Assert.assertEquals(songFromSuperPlaylist, getSongTitle);
+        List<String> songsFromSuperPlaylist = getSongTitleFromSuperPlaylist();
+        Assert.assertTrue(songsFromSuperPlaylist.contains(getSongTitle));
     }
 
-    private String getSongTitleFromSuperPlaylist() {
+    private List<String> getSongTitleFromSuperPlaylist() {
+        List<String> songs = new ArrayList<>();
+
+        List<WebElement> superPlaylistSongs = driver.findElements(By.xpath("//section[@id='playlistWrapper']//tr[@class='song-item']//td[@class='title']"));
+        if(superPlaylistSongs != null && !superPlaylistSongs.isEmpty()) {
+            for(WebElement each : superPlaylistSongs) {
+                songs.add(each.getText());
+            }
+        }
```

Comment:
I fixed your code, try this:

 ```
@Test
  public void addSongToPlaylist() {
    //login
    login();
    //add a song to the playlist

    clickRecentlyPlayed();
    String getSongTitle = getSongTitleTxt();
    clickSongFromList();
    clickAddToPlaylist();
    addToSuperPlaylist();
    clickOnSuperPlaylist();

    List<String> songsFromSuperPlaylist = getSongTitleFromSuperPlaylist();
    Assert.assertTrue(songsFromSuperPlaylist.contains(getSongTitle));
  }

  private List<String> getSongTitleFromSuperPlaylist() {
    List<String> songs = new ArrayList<>();

    List<WebElement> superPlaylistSongs = driver.findElements(
        By.xpath("//section[@id='playlistWrapper']//tr[@class='song-item']//td[@class='title']"));
    if (superPlaylistSongs != null && !superPlaylistSongs.isEmpty()) {
      for (WebElement each : superPlaylistSongs) {
        songs.add(each.getText());
      }
    }

    return songs;
  }

  private void clickOnSuperPlaylist() {
    WebElement superPlaylistLink = driver.findElement(By.xpath("//a[text()='super']"));
    superPlaylistLink.click();
  }

  private void addToSuperPlaylist() {
    WebElement superPlaylist = driver.findElement(
        By.xpath("//section[@id='recentlyPlayedWrapper']//li[contains(text(), 'super')]"));
    superPlaylist.click();
  }

  private String getSongTitleTxt() {
    WebElement getTitleTxt = driver.findElement(
        By.xpath("//section[@id='recentlyPlayedWrapper']//tr[@class='song-item']//td[@class='title']"));
    return getTitleTxt.getText();
  }

  private void clickAddToPlaylist() {
    WebElement addToPlaylist = driver.findElement(By.xpath("//button[contains(@title, 'Add selected songs')]"));
    addToPlaylist.click();
  }

  private void clickSongFromList() {

    WebElement songList = driver.findElement(
        By.xpath("//section[@id='recentlyPlayedWrapper']//tr[@class='song-item']"));
    songList.click();

  }

  private void clickRecentlyPlayed() {
    WebElement recentlyPlayed = driver.findElement(By.xpath("//li[@class='playlist recently-played']"));
    recentlyPlayed.click();
  }
}
```

![image](https://user-images.githubusercontent.com/104761482/200165873-b97e88e7-b9c4-46b7-b458-f1aacb5923b2.png)

I have added this method:

![image](https://user-images.githubusercontent.com/104761482/200165953-7dd04de4-28f7-4a82-a950-feb01b5686db.png)

![image](https://user-images.githubusercontent.com/104761482/200165964-13b96d76-8f4a-4495-b7ce-bc9406957b49.png)